src/runtime_src/xdp/profile/device/traceS2MM.cpp

```diff
@@ -60,6 +107,124 @@ void TraceS2MM::showProperties()
     ProfileIP::showProperties();
 }
 
+void TraceS2MM::showStatus()
+{
+  uint32_t reg_read = 0;
+  std::ostream *outputStream = (out_stream) ? out_stream : (&(std::cout));
+  (*outputStream) <<"--------------TRACE DMA STATUS-------------" << std::endl;
+  read(0x0, 4, &reg_read);
+  (*outputStream) << "INFO Trace dma control reg status : " << std::hex << reg_read << std::endl;
+  read(TS2MM_COUNT_LOW, 4, &reg_read);
+  (*outputStream) << "INFO Trace dma count status : " << reg_read << std::endl;
+  read(TS2MM_WRITE_OFFSET_LOW, 4, &reg_read);
+  (*outputStream) << "INFO Trace low write offset : " << reg_read << std::endl;
+  read(TS2MM_WRITE_OFFSET_HIGH, 4, &reg_read);
+  (*outputStream) << "INFO Trace high write offset : " << reg_read << std::endl;
+  read(TS2MM_WRITTEN_LOW, 4, &reg_read);
+  (*outputStream) << "INFO Trace written low : " << reg_read << std::endl;
+  read(TS2MM_WRITTEN_HIGH, 4, &reg_read);
+  (*outputStream) << "INFO Trace written high: " << reg_read << std::dec << std::endl;
+}
 
-}   // namespace xdp
+inline void TraceS2MM::parsePacketClockTrain(uint64_t packet, uint64_t firstTimestamp, unsigned mod, xclTraceResults &result)
```

Comment:
unsigned what ? this is a very bad habit used by almost everyone :-)  won't compile on windows.